charms/worker/k8s/templates/snap_installation.yaml

```diff
@@ -2,12 +2,10 @@
 # See LICENSE file for licensing details.
 
 amd64:
-  - name: k8s
-    install-type: store
-    classic: true
-    channel: 3372
+- name: k8s
+  install-type: store
+  revision: 3372
 arm64:
-  - name: k8s
-    install-type: store
-    classic: true
-    channel: 3368
+- name: k8s
+  install-type: store
+  revision: 3368
```

Comment:
@louiseschmidtgen the big changes here are a swap from `channel` to `revision` and dropping the `classic`